isovar/variant_read.py

```diff
@@ -148,8 +150,13 @@ def variant_read_from_single_read_at_locus(read, ref, alt):
                     read))
             return None
 
+    if sequence[read_pos_before + 1:read_pos_after] != alt:
```

Comment:
1) If the variant is at the end of the read then currently it doesn't get a ReadAtLocus. 

2) Deletions are the motivation for fussing about flanking nucleotides. Since the *ref* is missing from the read (by definition) I don't want to check other than the deletion happens in the right context. 

Do you have any ideas for something smarter I can check? Maybe I can get rid of all this logic about positions before/after the variant in all cases except deletions.